src/site/content/en/lighthouse-accessibility/aria-required-attr/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Ensure ARIA roles have required states and properties
+description: |
+  Learn about aria-required-attr audit.
+author: megginkearney
+web_lighthouse:
+  - aria-required-attr
+---
+
+[ARIA](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)
+roles and attributes help screen readers
+provide missing information about an element.
+For these roles and attributles to make sense,
+each ARIA `role` supports a specific subset of `aria-*` attributes
+(see [ARIA roles definitions](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)).
+Some ARIA roles have required attributes that describe the state of the element to screen readers. 
+Lighthouse reports a role's missing required states and properties:
+
+***TODO*** talk to Rob about this one. I can't seem to get a failed audit.
```

Comment:
I created an example here: https://concise-child.glitch.me/

`<div role=heading>` will fail because it requires an `aria-level` to specify the heading level.

If it's ok I'm leaving this TODO for you because it looks like you have a system down for doing consistent screen shots.

Reply:
Btw, a good way to find a failing example is to go to:
https://github.com/dequelabs/axe-core

Press the letter `T`. Start typing the audit name, then look for `test/integration/rules/<rule-name>.html`

Anything with the words "violation" or "fail" in the html will be a failing audit.
https://github.com/dequelabs/axe-core/blob/develop/test/integration/rules/aria-required-attr/required-attr.html#L39

Reply:
Very cool. New image in too.